src/root-config-guide/root-config-guide.component.js

```diff
@@ -0,0 +1,198 @@
+import React, { useContext } from "react";
+import PageHeader from "../shared/page-header.component";
+import { useCss } from "kremling";
+import { LocalStorageContext } from "../shared/use-local-storage-data.hook";
+import { SlackLink } from "../shared/links.component";
+import Code from "../shared/code.component";
+
+export default function HtmlFile(props) {
+  const scope = useCss(css);
+  const [copying, setCopying] = React.useState(false);
+  React.useEffect(() => {
+    if (copying) {
+      const timeoutId = setTimeout(() => {
+        setCopying(false);
+      }, 500);
+
+      return () => clearTimeout(timeoutId);
+    }
+  }, [copying]);
+  const { application } = useContext(LocalStorageContext);
+  const appUrls = window.importMapOverrides.getOverrideMap().imports;
+  if (!application)
+    return <p>You need to register at least one application!</p>;
+  const code = `
+<script type="systemjs-importmap">
+  {
+    "imports": {
+      "react": "https://cdn.jsdelivr.net/npm/react@16.13.0/umd/react.production.min.js",
```

Comment:
We shouldn't hard code react in here - single-spa-vue receives more traffic than single-spa-react does. Instead we should base the import map on the framework that they are using. This is why in the existing playground we ask them what framework they are using.